samcli/commands/local/lib/debug_context.py

```diff
@@ -5,6 +5,13 @@
 
 class DebugContext(object):
     def __init__(self, debug_port=None, debugger_path=None, debug_args=None):
+        """
+        Initialize the Debug Context with Lambda debugger options
+
+        :param tuple(int) debug_port: Collection of debugger ports to be exposed from a docker container
```

Comment:
Nit: Should this be `debug_ports` instead of `debug_port`

Reply:
Agreed. I thought about the same naming change. My original thought here was to persist the original CLI option `--debug-port` down to usages. Updated parameter names from `InvokeContext` down to `DebugContext`.

Reply:
Ya we can keep the option the same (`--debug-port`). This is just an internal object, so no concerns on the updates. 

Will do a re-review here shortly.